cadquery/cq.py

```diff
@@ -1346,6 +1346,48 @@ def move(self, xDist=0, yDist=0):
         newCenter = p + Vector(xDist, yDist, 0)
         return self.newObject([self.plane.toWorldCoords(newCenter)])
 
+    def slot(self, length, diameter, angle):
+        """
+        Creates a rounded slot for each point on the stack.
+
+        :param diameter: desired diameter, or width, of slot
+        :param length: desired end to end length of slot
+        :param angle: angle of slot in degrees, with 0 being along x-axis
+        :return: a new CQ object with the created wires on the stack
+
+        Can be used to create arrays of slots, such as in cooling applications:
+
+        result = cq.Workplane("XY").box(10,25,1).rarray(1,2,1,10).slot(8,1,0).cutThruAll()
+        """
+
+        def _makeslot(pnt):
+            """
+            Inner function that is used to create a slot for each point/object on the workplane
+            :param pnt: The center point for the slot
+            :return: A CQ object representing a slot
+            """
+
+            radius = diameter/2
+            centerPoint = self.plane.toWorldCoords(pnt)
+
+            p1 = centerPoint.add(Vector((-length/2) + radius, diameter/2))
```

Comment:
Why not using `+` operator?

Reply:
As in 
```
p1 = centerPoint + Vector((-length/2) + radius, diameter/2)
```
Honestly, I saw .add() used elsewhere so I just assumed `__add__` wasn't defined. I think changing to the `+` operator would be a good improvement in readability.

Reply:
It was added quite recently.